drivers/flash/Kconfig

```diff
@@ -42,6 +42,13 @@ config FLASH_PAGE_LAYOUT
 	help
 	  Enables API for retrieving the layout of flash memory pages.
 
+config FLASH_ERASE_VALUE
```

Comment:
Why is this feature conditional?  If drivers can't rely on the API call always being there then they can't guarantee to work on all devices where the erase value might vary.

Also this is misleading in that people will expect this to be the configured erase value, not a flag that enables an API to retrieve the actual erase value.  If it's going to be optional it should be `FLASH_ERASE_VALUE_API`.

Reply:
Generally disabling stuff helps to make footprint smaller - maybe in this case no worth as this will be just a few bytes per a driver instance.

So this can be always available IMO.
Otherwise I think it should be rather default 'y'